planck/PLKCommandLine.m

```diff
@@ -289,6 +291,7 @@ +(int)processArgsCount:(int)argc vector:(char * const *)argv
             printf("    -m ns-name, --main=ns-name Call the -main function from a namespace with\n");
             printf("                               args\n");
             printf("    -r, --repl                 Run a repl\n");
+            printf("    --quiet                    Run a repl in quiet mode.\n");
```

Comment:
A couple comments here:

1. It should probably indicate the short form as well (as in `-q, --quiet`)
2. It is not a main opt, so it should be moved up into the init opts section and perhaps the description revised to not indicate `Run a repl`